packages/palette-docs/content/docs/elements/buttons/Button.mdx

```diff
@@ -13,49 +14,52 @@ secondary-gray.
 Our secondary-outline is our default secondary button style. Avoid using this in
 the presence of inputs, as to not confuse it’s use.
 
-  <Box>
-    <Button width="33%">Purchase</Button>
-    <Spacer mb={1} />
-    <Button variant="secondaryOutline" width="33%">
-      Make offer
-    </Button>
-  </Box>
+<Box>
+  <Button width="33%">Purchase</Button>
+  <Spacer mb={1} />
+  <Button variant="secondaryOutline" width="33%">
+    Make offer
+  </Button>
+</Box>
 
 <Spacer my={2} />
 
-When needing to provide the ability to Show more content, we always use the secondary-outline style.
+When needing to provide the ability to Show more content, we always use the
+secondary-outline style.
 
 <Box>
-    <Button variant="secondaryOutline" width="33%">
-      Show more
-    </Button>
+  <Button variant="secondaryOutline" width="33%">
+    Show more
+  </Button>
 </Box>
 
 <Spacer my={2} />
 
 **Thinking usecases**
 
-The thinking state is used when there may be a delay in response time, ie, bidding, or following an artist.
+The thinking state is used when there may be a delay in response time, ie,
+bidding, or following an artist.
 
-  <Box>
-    <Button variant="primaryBlack" size="large" m={0.5} loading>
-      Show more
-    </Button>
-  </Box>
+<Box>
+  <Button variant="primaryBlack" size="large" m={0.5} loading>
+    Show more
+  </Button>
+</Box>
 
-Thinking inline should be used when the action taken reveals more content in the same context, ie, infinite scroll or expanding information.
+Thinking inline should be used when the action taken reveals more content in the
+same context, ie, infinite scroll or expanding information.
 
-  <Box>
-    <Button variant="secondaryOutline" width="33%">
-      Show more
-    </Button>
-  </Box>
+<Box>
+  <Button variant="secondaryOutline" width="33%">
+    Show more
+  </Button>
+</Box>
 
-  <Box>
-    <Button variant="secondaryGray" size="large" m={0.5} loading>
-      Show more
-    </Button>
-  </Box>
+<Box>
+  <Button variant="secondaryGray" size="large" m={0.5} loading>
+    Show more
+  </Button>
+</Box>
```

Comment:
Everything above here seems to be the result of VS Code doing a `formatOnSave` — with settings applicable to mdx files.

I'm not sure why these are only surfacing in this PR. Lmk if it looks like I've misconfigured something!